package.json

```
@@ -47,11 +47,11 @@
     "release:publish": "scripts/release-publish.sh"
   },
   "dependencies": {
-    "algoliasearch-helper": "^2.26.1",
+    "algoliasearch-helper": "^3.0.0",
```

```suggestion
    "algoliasearch-helper": "^2.28.1",
```

InstantSearch.js v3 needs an update with this too (however it should also be deduplicated if it doesn't)

---

Confirmed, we still get `Gzipped Size:  68.82 KB` with this update. We need to update instantsearch first right? 

```

/Users/nunomaduro/Algolia/vue-instantsearch/node_modules/algoliasearch-helper/package.json:
    1  {
    2:   "name": "algoliasearch-helper",
    3    "version": "2.28.1",
    4    "description": "Helper for implementing advanced search features with algolia",

/Users/nunomaduro/Algolia/vue-instantsearch/node_modules/instantsearch.js/node_modules/algoliasearch-helper/package.json:
    1  {
    2:   "name": "algoliasearch-helper",
    3    "version": "2.26.1",
    4    "description": "Helper for implementing advanced search features with algolia",
```

---

Seems that yarn did not correctly deduplicated the dependencies in the lockfile, I will fix this.